packages/code-du-travail-data/indexing/breadcrumbs.js

```diff
@@ -14,9 +14,15 @@ export function toBreadcrumbs({ label, position }) {
 
 export function createThemer(themes) {
   return function getTheme(slug) {
-    const theme = themes.find((theme) =>
-      theme.refs.some((ref) => ref.url.match(new RegExp(slug, "i")))
+    const referencedThemes = themes.filter((theme) =>
+      theme.refs.find((ref) => ref.url.match(new RegExp(slug, "i")))
     );
+    const theme = referencedThemes.reduce((majorTheme, currentTheme) => {
```

Comment:
I would sort theme and take the first item.
for sort, I would order theme by breadcrumb length (ie we take the most generic path) and fallback on the position when breadcrumb length is equal

Reply:
I'm not sure this will change anything since position is already some sort of hack to determine both the "depth" and the position (1 digit > depth 1, 2 digits = depth 2, 3 digits = depth 3 etc.) but yeah sorting and taking the first would definetely be more readable 👍